### ISSUE TITLE
Support article for "private" channels / revisit docs

### ISSUE THREAD

**Issue by mattheworiordan** [ably/docs#930]
Question from customer:

> Hey there, was wondering about having "private" channels. I need to have a secure way to publish to different clients while one can't read or subscribe to the other's data.
> What I thought while reading your docs was generating random hard-to-guess channel names per client and also encrypting them with per-client ciphers. Is this the way to go with Ably? Thanks.

I asked if he was referring to private channels because that is Pusher terminology:

> TBH, I'm new to realtime mesaging services so I don't have any established terminology yet.

So this indicates we need to better address this need in our docs, or at least improve our solution articles that are suggested automatically.

Here is his journey through the docs:

![screenshot_2020-08-01_07-53-51_pm](https://user-images.githubusercontent.com/43789/89108487-d842da00-d430-11ea-95d6-cd0b6620f778.png)

┆Issue is synchronized with this [Jira Task](https://ably.atlassian.net/browse/EDU-1755) by [Unito](https://www.unito.io)

**Comment by MarkWoulfeAbly** [comment]
@mattheworiordan do we currently offer the ability to create 'private channels'?

**Comment by mattheworiordan** [comment]
> do we currently offer the ability to create 'private channels'?

No, we don't offer "private" channels as such because every channel is both public and private, and access to channels is controlled via Ably's authentication, see https://www.ably.io/documentation/core-features/authentication#capabilities-explained.

Also, see the conversation with the customer at https://app.intercom.com/a/apps/ua39m1ld/inbox/inbox/search?assigneeIdentifier=all&q=Hey%20there%2C%20was%20wondering%20about%20having%20%22private%22%20channels
